### DISCUSSION TITLE
Is it a good way or lib to mock websocket?

### DISCUSSION THREAD

**Discussion by foreseaz** [vitest-dev/vitest#3280] [Q&A]
Is it a good way or lib to mock websocket that is recommended, thanks.

**Comment by akiomik** [comment]
Hi, I have created `vitest-websocket-mock` as a fork of `jest-websocket-mock`.

- repo: https://github.com/akiomik/vitest-websocket-mock
- npm: https://www.npmjs.com/package/vitest-websocket-mock
